projects/plugins/jetpack/tests/e2e/specs/recommendations.test.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * Internal dependencies
+ */
+import { step } from '../lib/env/test-setup';
+import RecommendationsPage from '../lib/pages/wp-admin/recommendations';
+
+describe( 'Jetpack Recommendations', () => {
+	it( 'Steps', async () => {
```

Comment:
This is a very long test, and the longer it is the more fragile it gets. I wonder if it's possible to break it into more smaller tests, maybe something like:

it('Check checkboxes')
it('Can enable Downtime monitoring')
it('Can enable Related posts')
etc.

The idea is to also make them independent of each other, so they can run in any order or if the first one fails the second one can run. I'm not very familiar with the tested scenario but after a quick check I believe this implies the manipulation of the application state in some way not through the UI. I'm not sure if/how that's possible.

Reply:
Yes. I thought about this a few times when I was writing the test. The problem is that you can't go back to a previous step once you completed it, which means that you can't easily test in isolation a specific step of the process without going through the previous one. That's why in the end, I decided to capture the whole flow in one big e2e test.

Reply:
I forgot to mention that this test only covers one flow which we consider the main one. But there are many other flows (enabling and disabling other combinations of features) that users might follow that we could capture in different tests.